DesktopEdge/Models/ZitiService.cs

```diff
@@ -121,13 +121,33 @@ public class ZitiService {
 
 		public Visibility TimerVisibility {
 			get {
-				return (TimeoutRemaining>-1&&TimeoutRemaining<1260) ? Visibility.Visible : Visibility.Collapsed;
+				return (TimeoutCalculated > -1&& TimeoutCalculated < 1260&& TimeoutCalculated > 0) ? Visibility.Visible : Visibility.Collapsed;

```

Comment:
```
TimeoutCalculated > -1&& TimeoutCalculated < 1260&& TimeoutCalculated > 0
```
This logic seems flawed. If the timeout is > 1260 this won't do anything? Also TimeoutCalculated > -1 is "nearly the same thing" as TimeoutCalculated > 0 .

Reply:
also all these 1260's are probably wrong and should be extracted to a constant somewhere. same for teh 1200's below

Reply:
These have been set to 1200, they were 1260 because the service would set at 20:32 or 20:12 etc. Since I manage the timer now it is set to 1200